client/src/test/java/com/vesoft/nebula/client/graph/data/TestData.java

```diff
@@ -142,7 +142,7 @@ public void testNode() {
         try {
             Node node = new Node(getVertex(new String("Tom")));
```

Comment:
I detect that this code is problematic. According to the [Performance (PERFORMANCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#performance-performance), [Dm: Method invokes inefficient new String(String) constructor (DM_STRING_CTOR)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#dm-method-invokes-inefficient-new-string-string-constructor-dm-string-ctor).
Using the java.lang.String(String) constructor wastes memory because the object so constructed will be functionally indistinguishable from the String passed as a parameter.  Just use the argument String directly.